### ISSUE TITLE
Bad Request Request has invalid param user when editing a submission

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#652]
**Description**
I can't edit anymore my Formie's submissions. I get a `Bad Request Request has invalid param user` error:
```
[-][1][jru4g9mvunikcflv39804mf028][error][yii\web\HttpException:400] yii\web\BadRequestHttpException: Request has invalid param user in /vendor/verbb/formie/src/controllers/SubmissionsController.php:1158
Stack trace:
#0 /vendor/verbb/formie/src/controllers/SubmissionsController.php(1038): verbb\formie\controllers\SubmissionsController->_getTypedParam()
#1 /vendor/verbb/formie/src/controllers/SubmissionsController.php(209): verbb\formie\controllers\SubmissionsController->_populateSubmission()
#2 [internal function]: verbb\formie\controllers\SubmissionsController->actionSaveSubmission()
#3 /vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#4 /vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams()
#5 /vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#6 /vendor/craftcms/cms/src/web/Application.php(287): yii\base\Module->runAction()
#7 /vendor/craftcms/cms/src/web/Application.php(596): craft\web\Application->runAction()
#8 /vendor/craftcms/cms/src/web/Application.php(266): craft\web\Application->_processActionRequest()
#9 /vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest()
#10 /htdocs/index.php(26): yii\base\Application->run()
#11 {main}
2021-11-23 10:48:47 [-][1][jru4g9mvunikcflv39804mf028][info][application] $_GET = [
    'p' => 'admin/formie/submissions/formHandle/9479'
    'site' => 'test'
]
```

This was previously working. I can't find exactly if this happened after a Craft or Formie update, or if it was after updating a form field handle...

**Steps to reproduce**
1. Edit a complete or incomplete submission
2. Get the `Bad Request Request has invalid param user` error

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.4.26
- Craft version: 3.7.21
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.26",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.27](https://github.com/verbb/formie/releases/tag/1.4.27)
